### PR TITLE
Add a condition to saphana ansible task to disable it

### PR DESCRIPTION
**Pull request by akrejcir**
**What this PR does / why we need it**:
Added 'when: 0 > 1' to the ansible task.

**Release note**:
```release-note
None
```

### CODE REVIEW

File: generate-templates.yaml [hunk 1]
```diff
@@ -40,20 +40,22 @@
...
-#      os: rhel8
-#      icon: rhel
-#      majorrelease: rhel8
-#      oslabels: "{{ rhel8_labels }}"
-#      osinfoname: "{{ oslabels[0] }}"
-#      cloudusername: cloud-user
+
+  # The SAP HANA template will not be generated
+  - name: Generate RHEL 8 SAP HANA template
+    when: 0 > 1 # This template is disabled
```
Comment (borod108) [inline generate-templates.yaml:46]: why don't we delete this and then pull it back from git? Is this for some kind of visibility?

**Comment by borod108** [comment]
I added a small question, but it is not critical.
/lgtm

File: generate-templates.yaml (hunk 1 above)
Comment (akrejcir) [reply generate-templates.yaml:46]: Yes. See comment on previous PR: https://github.com/kubevirt/common-templates/pull/389#discussion_r767539174

**Pull request merged**